_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,339 @@
+---
+layout:       pattern
+title:        Lists
+date:         2015-12-21 00:00:00
+
+categories:   patterns
+tags:
+- atomic
+- pattern
+- lists
+
+slug:         lists
+
+url_styles:     "patterns/_lists.scss"
+
+description: Lists are used to group related content in a number of settings and cases.
+
+info: Lists are used to group related content in a number of settings (e.g. ordered, unordered) and cases (e.g. definitions, links, etc.).
+---
+
+<h3 class="hd-6 example-set-hd">List (using different semantics)</h3>
+<div class="example-set">
+    <ul class="list">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ol class="list">
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+    </ol>
+
+    <div class="list">
+        <p class="item">Figs</p>
+        <p class="item">Oranges</p>
+        <p class="item">Plums</p>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Unstyled Lists</h3>
+<div class="example-set">
+    <ul class="list-unstyled">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Inline List</h3>
+<div class="example-set">
+    <ul class="list-inline">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Nested Lists</h3>
+<div class="example-set">
+    <ul>
+        <li class="item">
+            Apples
+
+            <ul>
+                <li>Red Delicious</li>
+                <li>Macintosh</li>
+                <li>Granny Smith</li>
+                <li>Honey Crisp</li>
+            </ul>
+        </li>
+        <li class="item">Apricots</li>
+        <li class="item">Bananas</li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ol>
+        <li class="item">Blueberries</li>
+        <li class="item">
+            Cherries
+
+            <ol>
+                <li>Bing</li>
+                <li>Balaton</li>
+                <li>Ulster</li>
+                <li>Maraschino</li>
+            </ol>
+        </li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+    </ol>
+</div>
+
+<h3 class="hd-6 example-set-hd">List with alignments</h3>
+<div class="example-set">
```

Comment:
Just a little concerned that the default does not have any visual list indicators (bullets or numbers).  It would be visually awkward to see bullets in the center example.  I just worry about someone who wanted visual bullets AND wanted to force a specific alignment.  Should we provide examples for both, or at least instructions for mixing classes?